src/StartingEnv.hs

```diff
@@ -279,6 +279,8 @@ dynamicModule =
     variadics =
       let f = addVariadicCommand . spath
        in [ f "array" commandArray "creates an array from a collection of elements." "(array 1 2 3) ; => [1 2 3]",
+            f "run-with-args" commandRunWithArgs "runs the built executable with arguments." "(run-with-args 1 2 3)",
+            f "run-exe-with-args" commandRunExeWithArgs "runs an executable with arguments." "(run-exe-with-args \"path-to-executable\" 1 2 3)",
```

Comment:
For consistency, should there also be a `run-exe`? But I guess that would just be `run-exe-with-args` without args, so that we could define

```clojure
(defndynamic run-exe [exe]
  (run-exe-with-args exe))
```

Reply:
Maybe even better would be to get rid of all the variants except `run-exe-with-args` and implement the rest in the carp side?

Reply:
That makes sense, since they’re all strictly less powerful. I guess the only thing we need to do for that is to find the compiler that should be used by default, then we’re good to go.

Reply:
Do you mean finding the resulting executable path?

Reply:
Yeah, exactly.